app.py

```diff
@@ -127,6 +131,7 @@ def user_leave_room(secret):
 @socketio.on('disconnect')
 def user_disconnect(): # ensure that eventlet is installed!!
     userObj = mdb.userDetails.find_one({'sid': request.sid})  # fetch user from db
+    socketio.emit('user_disconnected', room=userObj['room'])
```

Comment:
added this so user knows other user has disconnected (otherwise they can continue chatting to no one)